webserver-plugins/plugin-hibernate/src/main/java/org/webpieces/plugin/hibernate/metrics/DatabaseMetric.java

```diff
@@ -7,6 +7,9 @@
     DATABASE_ENTITY_UPDATES("webpieces/database/entity/updates"),
     DATABASE_ENTITY_INSERTS("webpieces/database/entity/inserts"),
 
+    EXECUTION_TIME("webpieces/database/execution/time", "/max"),
+    EXECUTION_COUNT("webpieces/database/execution/time", "/count"),
+
     TRANSACTION_TIME("webpieces/database/transaction/time", "/max"),
     TRANSACTION_TIME_COUNT("webpieces/database/transaction/time", "/count")
```

Comment:
Why use a different metric? If you don't want to publish to both, then remove the transaction ones.